runtime/v2/manager.go

```diff
@@ -213,186 +276,125 @@ func (m *TaskManager) startShim(ctx context.Context, bundle *Bundle, id string,
 	return shim, nil
 }
 
-// deleteShim attempts to properly delete and cleanup shim after error
-func (m *TaskManager) deleteShim(shim *shim) {
+// cleanupShim attempts to properly delete and cleanup shim after error
+func (m *ShimManager) cleanupShim(shim *shim) {
 	dctx, cancel := timeout.WithContext(context.Background(), cleanupTimeout)
 	defer cancel()
 
-	_, errShim := shim.delete(dctx, m.tasks.Delete)
-	if errShim != nil {
-		if errdefs.IsDeadlineExceeded(errShim) {
-			dctx, cancel = timeout.WithContext(context.Background(), cleanupTimeout)
-			defer cancel()
-		}
-		shim.Shutdown(dctx)
-		shim.Close()
-	}
+	_ = shim.delete(dctx)
```

Comment:
Shouldn't it log the error? It wasn't before though.

Reply:
let's keep this as is. i don't want to introduce new logic in this PR.